errors.json

```diff
@@ -29,5 +29,7 @@
     "bus.oidcBadIssuer": {"message": "OpenID issuer '{issuerId}' is not supported", "statusCode": 401},
     "bus.jsonRpcHttp": "JSON RPC returned HTTP error {code}",
     "bus.jsonRpcEmpty": "JSON RPC response without response and error",
-    "bus.noMeta": "$meta not passed to method: {method}"
+    "bus.noMeta": "$meta not passed to method: {method}",
+    "bus.mdns": "Multicast generic.",
+    "bus.mdns.resolver": "Multicast dns resolve. Can not resolve {service}"
```

Comment:
The convention is the error types to be consisted of 2 words. E.g. bus.mdnsResolver